app.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
```

Comment:
I assume you can use the latest `nodeThere is no reason to use `var` any longer, assuming you can use at least version `5.0` of node.